build/media-suite_compile.sh

```diff
@@ -861,7 +866,9 @@ if { { [[ $ffmpeg != "no" ]] &&
     do_patch "$_mingw_patches/0003-openal-not-32.mingw.patch"
     do_patch "$_mingw_patches/0004-disable-OSS-windows.patch"
     do_patch "https://0x0.st/zOfU.txt"
+    export CC=gcc.exe; export CXX=g++.exe
     do_cmakeinstall -DLIBTYPE=STATIC -DALSOFT_UTILS=OFF -DALSOFT_EXAMPLES=OFF
+    export CC=gcc; export CXX=g++
```

Comment:
```suggestion
    export CC=${CC%.exe}; export CXX=${CXX%.exe}
```

Might be better to use a temp variable?

Is setting .exe really necessary?

Reply:
Well, since you removed the lib/ccache from PATH, it doesn't really matter. Should be the same as `"${CC#ccache }"` above.